src/cli.cr

```diff
@@ -11,7 +11,7 @@ module Shards
           check                          - Verifies all dependencies are installed.
           init                           - Initializes a shard folder.
           install                        - Installs dependencies from `shard.lock` file.
-          list                           - Lists installed dependencies.
+          list [<options>]               - Lists installed dependencies.
```

Comment:
Maybe `list [--tree]` since there is only one option?

Reply:
Sure, fixed that.